### PR TITLE
Create 2016-09-16-minutes.markdown

### PR DESCRIPTION
**Pull request by ChadH1971**
Okay, here are the minutes for the first meeting with the new format agenda.

### CODE REVIEW

File: _posts/minutes/2016-09-16-minutes.markdown [hunk 1]
```diff
@@ -0,0 +1,93 @@
+#### BAKERSFIELD COLLEGE
+
+#### Computer Science Club
+
+#### 1801 Panorama Drive, Business Building, Room 5
+
+#### Bakersfield, California 93305
```
Comment (pathawks) [inline _posts/minutes/2016-09-16-minutes.markdown]: This top part might be better suited for the minutes template, if it will be the same each time.

Comment (ChadH1971) [inline _posts/minutes/2016-09-16-minutes.markdown]: There is much that will be the same each time, I'm just a little shaky on how to make a base template in github. Open to learning though.

Comment (asterwolf) [inline _posts/minutes/2016-09-16-minutes.markdown]: Chad, next time you see me on campus, ask me. I know how to make that change, I totally forgot to show you all the times I've seen you on campus.

Comment (pathawks) [inline _posts/minutes/2016-09-16-minutes.markdown]: #46

**Comment by pathawks** [comment]
meeting minutes september 16 2016

Here is how this page looks.

Notice that "Bakersfield College Computer Science Club" appears _twice_ at the top of the page. The date appears twice at the top of the page.


```html
<h4 id="bakersfield-college">BAKERSFIELD COLLEGE</h4>

<h4 id="computer-science-club">Computer Science Club</h4>

<h4 id="panorama-drive-business-building-room-5">1801 Panorama Drive, Business Building, Room 5</h4>

<h4 id="bakersfield-california-93305">Bakersfield, California 93305</h4>
<hr />

<h4 id="pm---300-pmbusiness-room-5">9/16/2016	1:00 p.m. - 3:00 p.m.	Business Room 5</h4>
```

This feels like an abuse of `<h4>` tags.

```html
<h3 id="icall-meeting-to-order">I.	CALL MEETING TO ORDER</h3>
<ul>
  <li>1:05 PM</li>
</ul>

<h3 id="iiroll-call">II.	ROLL CALL</h3>
<ul>
  <li>14/18 paid members present</li>
</ul>

<h3 id="iiiadoption-of-agenda">III.	ADOPTION OF AGENDA</h3>
<ul>
  <li>Agenda adopted without amendment</li>
</ul>
```

This might be better represented as a true ordered list, instead of `<h3>` tags separating `<ul>`s


```html
<ul>
  <li>A.	Ordering club shirts
    <ul>
      <li>i.	Take orders for new members
        <ul>
          <li>Send email asking for shirt sizes</li>
        </ul>
      </li>
      <li>ii.	ACTION: Vote for board members and advisors to have Polo Shirts
        <ul>
          <li>One opposed, discussion opened</li>
          <li>suggest modification as many expressed desire for Polos</li>
          <li>Ryan Wallace offered to front 50% of Polo cost.</li>
          <li>Deferred to next agenda</li>
        </ul>
      </li>
    </ul>
  </li>
  <li>B.	SoCal Code Camp OFFICIAL Sign up
```

This should clearly be an ordered list, rather than an unordered list.

At the top of the page, `BAKERSFIELD COLLEGE` appears in all caps. This is not how the name of the school is spelled, and is purely a presentation decision. This should probably be done in CSS instead of actually putting all caps in the page source.

---

That's what I've got. Take it or leave it: I'm not a club member anymore so it really isn't up to me at all 😄:+1:

**Comment by ChadH1971** [comment]
Pat, your input is always appreciated. As far as I'm concerned, those of us who put in the time to get this club started will always be members. I am working on learning jekyll and how to do things better. Question: If I were to prepare the post as .html instead of .markdown, would I be able to use the usual html tags? I agree with you that the outline would be better presented as a true ol, but I don't know yet how to work with markdown.

**Comment by pathawks** [comment]
> Question: If I were to prepare the post as .html instead of .markdown, would I be able to use the usual html tags?

Yes. Post as an HTML fragment inside a `.html` file and it will be dropped right into the `{{ content }}` of the template without any transformation.

If you’d like to learn more about Markdown, [this is a good starting point.](https://daringfireball.net/projects/markdown/syntax)

**Pull request merged**